### PR TITLE
Added db setup, data verification, and db clean to all tests

### PR DESCRIPTION
**Pull request by Lindessa**

### CODE REVIEW

File: Source/Test/Microsoft.Deployment.Site.Test/Microsoft-PSATemplateTests.cs [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+            var handlesInput = driver.FindElementByCssSelector("input[class='st-input au-target']");
+
+            handlesInput.SendKeys(handles);
+
+            HelperMethods.ClickButton("Validate");
+
+            var validated = driver.FindElementByClassName("st-validated");
+
+            Assert.IsTrue(validated.Text == "Successfully validated");
+        }
```
Comment (cdoroftei) [inline Source/Test/Microsoft.Deployment.Site.Test/Microsoft-PSATemplateTests.cs]: remove

File: Source/Test/Microsoft.Deployment.Site.Test/Microsoft.Deployment.Site.Test.csproj [hunk 2]
```diff
@@ -146,6 +149,12 @@
     </Content>
     <Content Include="Readme.txt" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (cdoroftei) [inline Source/Test/Microsoft.Deployment.Site.Test/Microsoft.Deployment.Site.Test.csproj]: was was this added?

File: Source/Test/Microsoft.Deployment.Site.Test/TestHelpers/HelperMethods.cs [hunk 3]
```diff
@@ -294,5 +370,107 @@ public static void CleanSubscription(string username, string password, string te
...
+                            "JOIN sys.dm_db_partition_stats s " +
+                            "ON t.object_id = s.object_id";
+
+            SqlConnection connection = new SqlConnection(sqlConnectionString);
+            SqlCommand command = new SqlCommand(script, connection);
+            try
+            {
+                connection.Open();
+                SqlDataReader reader = command.ExecuteReader();
+                reader.Read();
```
Comment (cdoroftei) [inline Source/Test/Microsoft.Deployment.Site.Test/TestHelpers/HelperMethods.cs]: is this needed?

Comment (raymondb1) [inline Source/Test/Microsoft.Deployment.Site.Test/TestHelpers/HelperMethods.cs]: Total rows can be retrieved in one statement, but the query above retrieves for all tables in the database (including those we don't want, like staging tables)

**Pull request merged**